components/src/main/res/layout/editable_list_items.xml

```diff
@@ -33,7 +33,8 @@
         android:importantForAccessibility="no"
         android:paddingHorizontal="0dp"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content" />
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical|end" />
 
     <View
         android:id="@+id/divider"
```

Comment:
Don't forget to add the padding to the divider now it's not added by the recycler view! :)